setup-nextcloud.php

```diff
@@ -76,8 +76,8 @@ static public function checkDependencies() {
 		$missingDependencies = array();
 
 		// do we have PHP 7.4.0 or newer?
```

Comment:
**nitpick:** Update the comment as well :)

Reply:
Useful Comments 300